processing/src/main/java/io/druid/segment/IndexIO.java

```diff
@@ -477,7 +476,12 @@ public QueryableIndex load(File inDir, ObjectMapper mapper) throws IOException
               metric,
               new ColumnBuilder()
                   .setType(ValueType.FLOAT)
-                  .setGenericColumn(new FloatGenericColumnSupplier(metricHolder.floatType, BYTE_ORDER))
+                  .setGenericColumn(new FloatGenericColumnSupplier(
```

Comment:
is this needed? this code path is used for legacy segments where nulls stuff is not supported

Reply:
the constr for FloatGenericColumnSupplier needs a nullValueBitmap, for legacy code we pass an empty bitmap as previously nulls were stored as empty strings.